Sources/UnleashProxyClientSwift/Poller.swift

```diff
@@ -176,8 +176,10 @@ public class Poller {
             
             self.createFeatureMap(features: json)
             if (self.ready) {
+                Printer.printMessage("Flag updated")
                 SwiftEventBus.post("update")
             } else {
+                Printer.printMessage("Initial flags fetched")
```

Comment:
@benjamin-es-hall Aren't these new prints?

Reply:
Ah yes sorry didn't release you were referring to these. However, they are only enabled when printToConsole is true and I would have thought when debugging you want as much information as possible? If it's too noisy it's within the users control to disable and in my testing when running this with short refresh interval it's the existing (no toggles changed) which is the noisiest unless you are rapidly changing toggles on the backend

Reply:
I can remove if needed though, the maintaining print state when calling update is far more important part since that did make me spend a significant amount of time debugging why it had appeared to stop updating.

These are similar that I was confused why when changing a toggle on backend I didn't see any log saying that was detected and only the "no toggles changed" log, but it was updating, just not reporting in a debug print mode

Reply:
On second thought, I think this is actually fine given that you can disable it and it's happening on the users device. I'll go ahead and merge this for now.

Reply:
@benjamin-es-hall Would you mind handling the conflict that arose after I merged your other PR?

Reply:
Awesome thanks! And yep let me get that fixed :)

Reply:
@FredrikOseberg That should be done now :)